src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -29,7 +29,7 @@ public enum ErrorCode {
   FILTER_UNRESOLVABLE("Unable to resolve the filter"),
 
   FILTER_MULTIPLE_ID_FILTER(
-      "Should only have one _id filter, document id cannot be restricted by more than one relation if it includes an Equal"),
+      "Cannot have more than one _id equals filter clause: use $in operator instead"),
```

Comment:
Would you get the same error message if you did something like `{ _id: { $in: ['foo', 'bar'] }, $or: [{ _id: { $in: ['baz', 'qux'] } }] }`?

Reply:
Hmmh. Good question; I do not know for sure... but I don't think you would.

@Yuqi-Du WDYT?

Reply:
I checked and yep, same error message if you do the following `Person.findOne({ _id: { $in: [john._id] }, $or: [{ _id: { $in: [bill._id] } }] })`

Maybe better to replace `use $in operator instead` with `use a single top-level $in operator instead`? Admittedly this is a bit of an edge case, but the error message looks a bit silly if you're already using `$in`.

Reply:
@vkarpov15 agreed, this is still not great. Would you mind filing another issue, referencing this PR, for further work? (something stating that only one `$eq` or `$in` clause can reference `_id` field)

I try not to re-open issues when there may be a release containing (partial) fix, release note generation tends to get things wrong in such cases.

Reply:
@tatu-at-datastax done #950

Reply:
Thank you @vkarpov15 !